lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -115,12 +115,32 @@
                     return DateUtils.localize(context);
                 },
 
+                getCertificatePriceString: function(run) {
+                    var verifiedSeat, currency;
+                    if ('seats' in run && run.seats.length) {
+                        // eslint-disable-next-line consistent-return
+                        verifiedSeat = _.filter(run.seats, function(seat) {
+                            if (['verified', 'professional', 'credit'].indexOf(seat.type) >= 0) {
+                                return seat;
+                            }
+                        })[0];
+                        currency = verifiedSeat.currency;
+                        if (currency === 'USD') {
```

Comment:
for the other currencies we just have the name. 

It would be weird to put something like USD10. This is basically only handling USD and adding a relatively sane fallback behavior for other currencies until we know how we want to deal with them